vortexor/src/cli.rs

```diff
@@ -51,6 +52,28 @@ fn get_default_tpu_coalesce_ms() -> &'static str {
     coalesce
 }
 
+fn parse_http_url(input: &str) -> Result<Url, String> {
```

Comment:
This and websocket version should be one function taking in the schemes to match against as argument.

Reply:
done